Dijkstras-Algo.ipynb

```diff
@@ -0,0 +1,166 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "G = nx.DiGraph()\n",
+    "G.add_edge(0, 1,weight=2.0)\n",
+    "G.add_edge(1, 2,weight=1.0)\n",
+    "G.add_edge(3, 4,weight=4.0)\n",
+    "G.add_edge(1, 5,weight=6.0)\n",
+    "G.add_edge(3, 6,weight=7.0)\n",
+    "G.add_edge(4, 6,weight=2.0)\n",
+    "G.add_edge(5, 6,weight=2.0)\n",
+    "draw_graph(G)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
add comment/markdown about adj matrix in notebook

Reply:
on it